README.md

```diff
@@ -173,9 +172,29 @@ You can then set the active environment by passing the `ENV_NAME` environment va
 docker run -e ENV_NAME=env2 my_multi_conda_image
 ```
 
-### Changing the user
+### Changing the user id or name
+
+Prior to June 30, 2021, the image defaulted to running as root. Now it defaults to running as the non-root user `mambauser` (defined during build as the value of `$MAMBA_USER`). Micromamba-docker can be run as any user by passing the `docker run ...` command the `--user=UID:GID` parameters. Running with `--user=root` is supported.
+
+There are two supported methods for changing the username:
+
+1. If rebuilding this image from scratch, the default username `mambauser` can be adjusted by passing `--build-arg MAMBA_USER=new-username` to the `docker build` command.
 
-Prior to June 30, 2021, the image defaulted to running as root. Now it defaults to running as the non-root user micromamba. Micromamba-docker can be run as any user by passing the `docker run ...` command the `--user=UID:GID` parameters. Running with `--user=root` is supported.
+2. When building an image `FROM` an existing micromamba image,
+
+    ```Dockerfile
+    FROM mambaorg/micromamba:0.19.1
+    ARG NEW_MAMBA_USER=new-username
+    USER root
+    RUN usermod "--login=${NEW_MAMBA_USER}" "${MAMBA_USER}" && \
+        groupmod "--new-name=${NEW_MAMBA_USER}" "${MAMBA_USER}" && \
+        mv "/home/${MAMBA_USER}" "/home/${NEW_MAMBA_USER}" && \
+        # Disables the consistency check in _entrypoint.sh:
+        rm "/etc/arg_mamba_user" && \
```

Comment:
Why are you removing the record of `MAMBA_USER` at build time rather than updating it?

Reply:
The way I see it, by executing these commands in the Dockerfile, the author is taking responsibility for the `MAMBA_USER` account, so there is no longer anything to be checked.

Reply:
My understanding of your motivation is to prevent confusion if someone doesn't realize that modifying `MAMBA_USER` doesn't magically change the username. Thus my goal is to trigger an error in this case and point the author to the relevant documentation.

Or do you see some further consistency checks as within scope for the micromamba-docker project?

Reply:
@wholz, I just pushed a commit where I echo the new username to `/etc/arg_mamba_user`. Does this help?